src/HeaderCell.tsx

```diff
@@ -43,17 +43,29 @@ export const resizeHandleClassname = css`
 const cellDraggableClassname = 'rdg-cell-draggable';
 
 const cellDragging = css`
-  opacity: 0.5;
+  @layer rdg.HeaderCell {
+    background-color: var(--rdg-header-draggable-background-color);
```

Comment:
Kept the same style as cell over for now. We cannot use opacity as we are using a custom drag image